docs/content/guides/author-apps/azure/howto-azure-container-instances/snippets/app.bicep

```diff
@@ -0,0 +1,97 @@
+//ENVIRONMENT
+extension radius
+
+resource env 'Applications.Core/environments@2023-10-01-preview' = {
+  name: 'aci-demo'
+  properties: {
+    compute: {
+      kind: 'aci'
+      // Replace value with your resource group ID
+      resourceGroup: '/subscriptions/<>/resourceGroups/<>'
+      identity: {
+        kind:'userAssigned'
+        // Replace value with your managed identity resource ID
+        managedIdentity: ['/subscriptions/<>/resourceGroups/<>/providers/Microsoft.ManagedIdentity/userAssignedIdentities/<>']
+      }
+    }
+    recipes: {
+      'Applications.Datastores/redisCaches': {
+        default: {
+          templateKind: 'bicep'
+          plainHttp: true
+          templatePath: 'ghcr.io/radius-project/recipes/azure/rediscaches:latest'
+        }
+      }
+    }
+    providers: {
+      azure: {
+        // Replace value with your resource group ID
+        scope: '/subscriptions/<>/resourceGroups/<>'
+      }
+    }
+  }
+}
+//ENVIRONMENT
+
+//APPLICATION
+resource app 'Applications.Core/applications@2023-10-01-preview' = {
+  name: 'demo-app'
+  properties: {
+    environment: env.id
```

Comment:
if we're splitting up the env and app deployments, we might need to update `env.id` to `environment` (with `environment` being added as a parameter to the Bicep file. 

if when we deploy app.bicep a second time, we're keeping the environment resource definition in the Bicep file, then it doesn't matter

Reply:
thanks, updated